_rules/SC2-4-6-descriptive-labels.md

```diff
@@ -0,0 +1,125 @@
+---
+name: Label is descriptive
+test_type: atomic
+
+description: |
+  This rule checks that labels describe the purpose of form field elements.
+
+success_criterion:
+- 2.4.6 # Headings and labels
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+
+authors:
+- Dagfinn Rømen
+- Geir Sindre Fossøy
+---
+
+## Test Procedure
+
+### Applicability
+
+This rule applies to any `label` element or HTML element referenced by aria-labelledby that is 
```

Comment:
The position in which the delimiter "HTML" appears makes the context of the former reference to `label` ambiguous. I think it should instead say:

> This rule applies to any HTML `label` element or other element referenced by `aria-labelledby` that is

Reply:
Good point. Updated.